tests/mmtf_tests.cpp

```diff
@@ -7,10 +7,11 @@
 // NOTE!!! Margin is set to 0.00001
 // If we ever get more specific data this may need to be altered!
 template <typename T>
-bool approx_equal_vector(const T& a, const T& b) {
+bool approx_equal_vector(const T& a, const T& b, float eps = 0.00001) {
 	if (a.size() != b.size()) return false;
-	for (std::size_t i=0; i<=a.size(); ++i) {
-		if (a[i] != Approx(b[i]).margin(0.00001)) return false;
+	// Please be careful! '<=' is not a good idea!
```

Comment:
It was just a miss-type/unintentional you can delete this comment, or leave it as a reminder to all rosetta developers :p

Reply:
I'll remove it in my next PR that adds basic CI support :smile: